qscript.template

```diff
@@ -58,8 +58,9 @@ echo "INFO: $THIS: LD_LIBRARY_PATH : $LD_LIBRARY_PATH"
 #----------------------------------------------------------------------------
 #               W R I T E   J O B   P R O P E R T I E S
 #----------------------------------------------------------------------------
-# job properties (TODO: Add json propertes)
-
+if [[ -e %adcircdir%/adcirc.bin.buildinfo.json ]]; then
+   cp %adcircdir%/adcirc.bin.buildinfo.json . 2>> $SYSLOG
+fi
```

Comment:
does this need to get recorded in `run.properties`?

Reply:
yes, I will do that, and wrap the `cp` target in double quotes